src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -113,16 +133,64 @@ export default class CellAction extends React.Component<Props, State> {
         }
         query[negation].push(`${value}`);
         break;
+      case Actions.SHOW_GREATER_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`>${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort descending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'desc')!]);
+
+        break;
+      }
+      case Actions.SHOW_LESS_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`<${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort ascending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'asc')!]);
+
+        break;
+      }
+      case Actions.TRANSACTION: {
+        const maybeProject = projects.find(project => {
+          return project.slug === dataRow.project;
+        });
+
+        const projectID = maybeProject
+          ? [maybeProject.id]
+          : eventView.project.map(id => String(id));
```

Comment:
What happens if the eventView has multiple projects?

Reply:
I think this should be `undefined` if there is no project id. https://github.com/getsentry/sentry/pull/19230/commits/5152bab8f6649d52c43b6096cfaf6243215ccb55

@bowencai8 can you confirm?

Reply:
Bowen is fine with making this `undefined`